dev_utils/compose-no-tls.yml

```diff
@@ -75,7 +86,6 @@ services:
     image: python:3.10-slim
     volumes:
       - ./mockoidc/mockoidc.py:/mockoidc.py
-      - ./certs:/certs
```

Comment:
Not saying this is wrong, but I'd appreciate seeing the rationale for moving these to unencrypted.

(I don't think it's a real problem but have a slight worry that having the automated tests without https here could miss something.)

Reply:
No, I got confused and mixed the changes up.